doc/man3/CRYPTO_get_ex_new_index.pod

```diff
@@ -130,9 +130,9 @@ the same callback handles different types of exdata.
 dup_func() is called when a structure is being copied.  This is only done
 for B<SSL> and B<SSL_SESSION> objects.  The B<to> and B<from> parameters
 are pointers to the destination and source B<CRYPTO_EX_DATA> structures,
-respectively.  The B<srcp> parameter is a pointer to the source exdata.
-When the dup_func() returns, the value in B<srcp> is copied to the
-destination ex_data.  If the pointer contained in B<srcp> is not modified
+respectively.  The B<*srcp> parameter is a pointer to the source exdata.
```

Comment:
aehm, this should probably refer to from_d, as the parameter is called in the prototype..